# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/chromium/hotfix-rust-cbor.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- chromium-153.0.8010.36/components/cbor/BUILD.gn.vanilla	2026-09-10 19:20:48.966204315 +0200
+++ chromium-153.0.8010.36/components/cbor/BUILD.gn	2026-09-10 19:22:49.694535134 +0200
@@ -12,7 +12,7 @@
   # TODO(crbug.com/535682335): Remove `USE_CBOR_RUST` buildflag entirely,
   # unconditionally enable Rust CBOR parser, and drop `is_cronet_build` check
   # once Cronet supports Crubit dependencies.
-  flags = [ "USE_CBOR_RUST=!$is_cronet_build" ]
+  flags = [ "USE_CBOR_RUST=$enable_cpp_api_from_rust" ]
 }
 
 component("cbor") {
@@ -35,7 +35,7 @@
     "//base",
   ]
 
-  if (!is_cronet_build) {
+  if (enable_cpp_api_from_rust) {
     public_deps = [
       "//build/rust/crubit",
       "//components/cbor/rust:cbor_rust_bindings",
@@ -61,7 +61,7 @@
     "//testing/gtest",
     "//third_party/fuzztest",
   ]
-  if (!is_cronet_build) {
+  if (enable_cpp_api_from_rust) {
     deps += [ "//components/cbor/rust:cbor_rust_unittests" ]
   }
 
